src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1229,3 +1230,42 @@ def _configure_security_policy_storage_params(arg_ctx):
                    id_part='name')
 
         c.argument('resource_group_name', arg_type=resource_group_name_type)
+
+    ###############################################
+    #             sql instance failover-group     #
+    ###############################################
+
+    with self.argument_context('sql instance-failover-group') as c:
+        c.argument('instance_failover_group_name',
+                   options_list=['--name', '-n'],
+                   help="The name of the Instance Failover Group")
+
+        c.argument('failover_group_name',
+                   options_list=['--name', '-n'],
+                   help="The name of the Instance Failover Group")
```

Comment:
Why is there both instance_failover_group_name and failover_group_name? Please fix swagger that there is only one of these.